integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSource.scala

```diff
@@ -334,6 +334,11 @@ object CarbonSource {
     properties.foreach(e => map.put(e._1, e._2))
     map.put("tablepath", identifier.getTablePath)
     map.put("dbname", identifier.getDatabaseName)
+    val LOGGER = LogServiceFactory.getLogService(this.getClass.getCanonicalName)
+    if (map.containsKey("tableName")) {
+      LOGGER.warn("tableName is useless in option when create table")
```

Comment:
change to `tableName is not required in options, ignoring it`

Reply:
ok,done